repository.eselect.5.in

```diff
@@ -0,0 +1,169 @@
+.\" -*- coding: utf-8 -*-
+.\" Copyright 2019 Till Schäfer
+.\" Distributed under the terms of the 2-clause BSD license
+.\"
+.TH repository.eselect 5 "31 January 2019" "Gentoo Linux" "eselect"
+.SH NAME
+\fBrepository.eselect\fP - Manage repos.conf via eselect
+.SH SYNOPSIS
+.nf
+.fam C
+\fBeselect\fP \fBrepository\fP [\fBhelp\fP|\fBusage\fP|\fBversion\fP]
+\fBeselect\fP \fBrepository\fP \fBadd\fP \fIname\fP \fIsync-type\fP \fIsync-uri\fP
+\fBeselect\fP \fBrepository\fP \fBdisable\fP [\fB-f\fP] (\fIname\fP|\fIindex\fP)
+\fBeselect\fP \fBrepository\fP \fBenable\fP (\fIname\fP|\fIindex\fP)
+\fBeselect\fP \fBrepository\fP \fBlist\fP [\fB-i\fP]
+\fBeselect\fP \fBrepository\fP \fBremove\fP [\fB-f\fP] (\fIname\fP|\fIindex\fP)
+
+.SH DESCRIPTION
+\fBrepository.eselect\fP is an eselect module for manipulating /etc/portage/repos.conf entries to third party repositories (also known as overlays). This supersedes the need for app-admin/layman to list, configure, and handle synchronizations of alternate repositories except for those which the package manager does not (eg. mercurial, bazaar and g-sorcery in portage).
```

Comment:
Could you wrap at 72 chars? Also it controls entries to *all* repositories, including ::gentoo. Also, I don't think listing layman is exactly the way to describe eselect-repo.